app/controllers/application_controller.rb

```diff
@@ -184,6 +184,21 @@ def es_query(klass, query, filters)
                         order: format_order).paginate(page: page_number, per_page: per_page_number)
   end
 
+  def pg_search_projects(term)
+    ProjectSearchQuery.new(
+      term,
+      platforms: current_platforms,
+      languages: current_languages,
+      keywords: current_keywords,
+      licenses: current_licenses,
+      sort: format_sort
+    ).results
+  end
+
+  def pg_search_projects_enabled?
+    params.key?("force_pg") || Rails.application.config.pg_search_projects_enabled
```

Comment:
Probably only want to allow the param to be passed if the user is admin/internal?